lib/config/presets/codecommit/index.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { TextEncoder } from 'web-encoding';
+import { Platform, setPlatformApi } from '../../../modules/platform';
+import { PRESET_DEP_NOT_FOUND } from '../util';
+import * as codeCommit from '.';
+
+jest.unmock('../../../modules/platform');
+jest.mock('@aws-sdk/client-codecommit');
```

Comment:
use AWS sdk mock instead, check util/s3 tests

Reply:
i checked those already before i started, i tried to make them work, but they just dont mock for me, 
after 4 hours of frustration, i decided to look at their implementation in AWS code, and i copied the internal implementation and it now mocks and works.

i'm not an expert on Jest, but those examples just dont work for codecommit

i will give it another try, but i'm not sure what i was missing :'(

Reply:
i'm able to use import {mockClient} from 'aws-sdk-client-mock';

Reply:
Please use `aws-sdk-client-mock` it should work.

You need to call `mockClient(CodeCommitClient)` not ``mockClient(CodeCommit)`

https://github.com/m-radzikowski/aws-sdk-client-mock

Reply:
i already did that yesterday

Reply:
https://github.com/renovatebot/renovate/blob/e898a7d286018581d50e39d5a4c26b1ca432e450/lib/modules/platform/codecommit/index.spec.ts

i dont get it, see the new file, why is this not "Outdated" ?